src/app/pages/home/home.component.html

```diff
@@ -29,4 +30,12 @@
       </p>
     </mat-card-content>
   </mat-card>
+  <mat-card class="privacy-card">
+    <mat-card-content class="privacy-card">
+      <button class="privacy-button" routerLink="/privacy-policy">
+        <mat-icon>privacy_tip</mat-icon>
+      </button>
+    </mat-card-content>
+  </mat-card>
```

Comment:
Should we remove this an plan to add the link to the privacy policy in the about page branch?

Reply:
Th privacy policy should be linked on the main page (though this link is incorrect)

Reply:
@audreylemeur - Right, it should be linked from the main page, but I think @floogulinc is just saying that we could remove the material card and icon stuff in this branch. Perhaps we should replace it with a link in this branch, though, rather than waiting until the About Page branch is done. Thoughts?

Reply:
Would it be ok to remove the link here and have someone else add it in the about branch? I don't want to step on anyone's toes since I don't know how it should look with the About link.